maps_dashboards/public/components/layer_config/documents_config/document_layer_source.tsx

```diff
@@ -303,6 +309,16 @@ export const DocumentLayerSource = ({
             filters={selectedLayerConfig.source.filters ?? []}
             onFiltersUpdated={onFiltersUpdated}
           />
+          <EuiSpacer />
+          <EuiFormRow>
+            <EuiSwitch
```

Comment:
I found out that switch is preferred if an action is performed immediately. In this case, the update happens after user update layer config. Shall we change this control to checkbox?

Reply:
In the screenshot you shared, it's a switch component, shall we make it consistent?

Reply:
Yeah. I shared screenshot from existing component ( i thought i mentioned in text), but i received this feedback from UX designer for tooltip. We also changed to checkbox for tooltip too.

Reply:
That's clear, I will update.